.github/workflows/create-test-plan.py

```diff
@@ -556,6 +556,9 @@ def spec_to_job(spec: JobSpec, key: str, trackmem_symbol_names: bool) -> JobDeta
             job.apt_packages = []
             job.apk_packages = ["cmake", "gmp", "mpc1", "mpfr4", "ninja", "pkgconf", "git", ]
             job.cmake_toolchain_file = "${PS2DEV}/ps2sdk/ps2dev.cmake"
+            job.cmake_arguments.extend((
```

Comment:
Don't configure defaults in the ci script.
Rather, do it in the cmake script.
https://github.com/libsdl-org/SDL/blob/a47b62d1fe6eb8e2f9fd5a2b015f849fb9972255/CMakeLists.txt#L137-L141

Reply:
👍

Reply:
Now it´s working properly.